core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -78,28 +78,30 @@ fun LpPostCard(
     userPhoto: Painter,
     timeAgo: String,
     messagePost: String,
-    sizeWidth: Float  = 1F,
+    sizeWidth: Float = 1F,
     onCommentClick: () -> Unit,
     onFavoriteClick: () -> Unit
-){
+) {
     Card(
         modifier = modifier
             .fillMaxWidth(sizeWidth)
```

Comment:
This modifier property `fillMaxWidth(sizeWidth)` shouldn't be on this modifier, the function that implement this component is the only one that defines this kind of sizes. So you could remove this property